src/status.js

```diff
@@ -0,0 +1,139 @@
+// import { clearCont } from './index.js';
```

Comment:
- [ ]  Please kindly remove commented code from you code base as this makes you work dirty and less readable.

Reply:
Thank you @Mihndim2020 for your suggestions! I removed these comments from my code.